src/java/htsjdk/tribble/gelitext/GeliTextCodec.java

```diff
@@ -1,112 +0,0 @@
-/*
```

Comment:
is this used in IGV?  This needs to be checked before you can remove things in tribble.